html/portfolio.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Emeka Portfolio Works</title>
+    <link rel="stylesheet" href="/css/portfolio.css">
```

Comment:
**[Optional]**

- [ ] Kindly fix the stylesheet link because in production it is broken 😨 Kindly fix it by putting two dots at the start of the link 👍 as shown below ⬇️

 `href="../css/portfolio.css"`